### DISCUSSION TITLE
PageSpeed Insights doesn't return full-page-screenshot since move to LH 10

### DISCUSSION THREAD

**Discussion by goynang** [GoogleChrome/lighthouse#14900] [General] [answered]
Now that Lighthouse v10 has moved the location of the `full-page-screenshot` data to the top level of the JSON response (and out of the `audits` section) it is no longer returned by PageSpeed Insights at all (tested using the PSI API).

Does anyone know if it is deliberately missing from PageSpeed Insights or am I just missing something?

Thanks.

**Comment by adamraine** [comment, marked as answer]
Should be fixed https://github.com/GoogleChrome/lighthouse/issues/14886, though the fix may not be in production yet. @paulirish can you confirm that this is the case?

**Reply by goynang** [reply]
Thanks... I should have found that issue but my search-fu let me down.
